### ISSUE TITLE
Not working with ArkInventory?

### ISSUE THREAD

**Issue by Obsurd** [p3lim-wow/Molinari#32]
- What version are you using? 80000.58-Release

Not 100% sure if this is a Molinari bug or with ArkInventory.  After the patch yesterday, this addon is no longer working with ArkInventory loaded.  I unload it and it works.  I use another version of AI that this WAS working with just fine prior to the patch and it still doesn't work.  :(

**Comment by tordenflesk** [comment]
Same issue with Cargbags_Nivaya:

```
44x Molinari\Molinari-80000.58-Release.lua:91: Action[SetPoint] failed because[SetPoint would result in anchor family connection]: attempted from: Molinari:SetAllPoints.
[C]: in function `SetAllPoints'
Molinari\Molinari-80000.58-Release.lua:91: in function `Apply'
Molinari\Molinari-80000.58-Release.lua:125: in function <Molinari\Molinari.lua:102>
[C]: ?
...rfaceTradeSkillMaster\Private\TooltipLib.lua:85: in function <...rfaceTradeSkillMaster\Private\TooltipLib.lua:82>
[C]: ?
[C]: ?
[C]: ?
[C]: ?
[C]: ?
[C]: ?
[C]: ?
...
...eBartender4\libs\AceHook-3.0\AceHook-3.0-8.lua:95: in function <...eBartender4\libs\AceHook-3.0\AceHook-3.0.lua:87>
[C]: ?
[C]: ?
[C]: ?
...rfaceTradeSkillMaster\Private\TooltipLib.lua:95: in function <...rfaceTradeSkillMaster\Private\TooltipLib.lua:93>
[C]: ?
[C]: ?
[C]: in function `SetBagItem'
FrameXML\ContainerFrame.lua:1403: in function `ContainerFrameItemButton_OnEnter'
[string "*:OnEnter"]:1: in function <[string "*:OnEnter"]:1>

Locals:
(*temporary) = Molinari {
 0 = <userdata>
 UpdateModifier = <function> defined @Molinari\Molinari.lua:97
 sparkles = <table> {
 }
 Apply = <function> defined @Molinari\Molinari.lua:58
}
(*temporary) = NivayaSlot160 {
 timeSinceUpgradeCheck = 0
 PostOnLoad = <function> defined @FrameXML\ItemButtonTemplate.lua:169
 BattlepayItemTexture = <unnamed> {
 }
 IconOverlay = <unnamed> {
 }
 BagStaticTop = <unnamed> {
 }
 bagID = 4
 GetItemContextMatchResult = <function> defined @FrameXML\ContainerFrame.lua:1464
 IconBorder = <unnamed> {
 }
 newitemglowAnim = <unnamed> {
 }
 icon = NivayaSlot160IconTexture {
 }
 container = NivayacBniv_Armor {
 }
 UpdateTooltip = <function> defined @FrameXML\ContainerFrame.lua:1376
 ExtendedOverlay = <unnamed> {
 }
 RegisterCallback = <function> defined @FrameXML\ItemButtonTemplate.lua:175
 UnregisterCallback = <function> defined @FrameXML\ItemButtonTemplate.lua:182
 UpdateItemContextMatching = <function> defined @FrameXML\ItemButtonTemplate.lua:211
 NewItemTexture = <unnamed> {
 }
 Cooldown = NivayaSlot160Cooldown {
 }
 SplitStack = <function> defined @FrameXML\ContainerFrame.lua:1100
 BagStaticBottom = <unnamed> {
 }
 ItemContextOverlay = <unnamed> {
 }
 PostOnShow = <function> defined @FrameXML\ItemButtonTemplate.lua:189
 flash = <unnamed> {
 }
 itemContextChangedCallbackIsSet = true
 ExtendedSlot = <unnamed> {
 }
 ExtendedOverlay2 = <unnamed> {
 }
 itemContextMatchResult = 3
 Icon = NivayaSlot160IconTexture {
 }
 count = 1
 JunkIcon = <unnamed> {
 }
 SetMatchesSearch = <function> defined @FrameXML\ItemButtonTemplate.lua:202
 UpdateItemContextOverlay = <function> defined @FrameXML\ItemButtonTemplate.lua:222
 slotID = 8
 BottomString = <unnamed> {
 }
 TopString = <unnamed> {
 }
 Border = <unnamed> {
 }
 Quest = NivayaSlot160IconQuestTexture {
 }
 itemContextChangedCallback = <function> defined @FrameXML\ItemButtonTemplate.lua:170
 0 = <userdata>
 Count = NivayaSlot160Count {
 }
 PostOnHide = <function> defined @FrameXML\ItemButtonTemplate.lua:198
 flashAnim = <unnamed> {
 }
 GetMatchesSearch = <function> defined @FrameXML\ItemButtonTemplate.lua:207
 UpgradeIcon = <unnamed> {
 }
 searchOverlay = NivayaSlot160SearchOverlay {
 }
}

```

**Comment by Mythris29** [comment]
Also having the same issues after the wow 8.2 patch

**Comment by Obsurd** [comment]
Fix posted on Curse, at least in regards to AI.  Assume it's the same issue being had with others.

Add the following between line 90 and 91, making this the new line 91.

self:ClearAllPoints()

**Comment by p3lim** [comment]
Fixed in #33.
